spec/index.bs

```diff
@@ -7058,17 +7091,19 @@ dictionary GPURenderPassDepthStencilAttachment {
     Given a {{GPURenderPassDepthStencilAttachment}} |this| the following validation
     rules apply:
 
-    1. |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
+    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
         format=].
-    1. |this|.{{GPURenderPassDepthStencilAttachment/view}} must be a view of a
+    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must be a view of a
         single [=texture subresource=].
-    1. |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
+    - |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
         must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
-    1. |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} is `true`,
-        |this|.{{GPURenderPassDepthStencilAttachment/depthLoadValue}} must be
-        {{GPULoadOp/"load"}} and |this|.{{GPURenderPassDepthStencilAttachment/depthStoreOp}}
-        must be {{GPUStoreOp/"store"}}.
-    1. |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}} is `true`,
+    - If |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}
+        contains both depth and stencil aspects (See [[#depth-formats|depth-stencil formats]]):
+        - |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} must be equal to |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}}
+    - If |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} is `true`:
+        - |this|.{{GPURenderPassDepthStencilAttachment/depthLoadValue}} must be {{GPULoadOp/"load"}}.
+        - |this|.{{GPURenderPassDepthStencilAttachment/depthStoreOp}} must be {{GPUStoreOp/"store"}}.
```

Comment:
Interesting, isn't this going to cause tilers to load the memory and then store it back? I wonder if we could rely on a discard not touching the memory so the content would be kept. Seems unlikely to work all the time though.

Reply:
This text is what we already have in the spec, unaffected by the PR.
Also see https://github.com/KhronosGroup/Vulkan-Docs/issues/1345 if you have time to read through it ;)